Source/BoundaryConditions/WarpXFieldBoundaries.cpp

```diff
@@ -131,6 +135,8 @@ void WarpX::ApplyBfieldBoundary (const int lev, PatchType patch_type, DtType a_d
     if (lev == 0) {
         if (a_dt_type == DtType::FirstHalf) {
             if(::isAnyBoundary<FieldBoundaryType::Absorbing_SilverMueller>(field_boundary_lo, field_boundary_hi)){
+                auto Efield_fp = m_fields.get_mr_levels_alldirs("Efield_fp",max_level);
+                auto Bfield_fp = m_fields.get_mr_levels_alldirs("Bfield_fp",max_level);
```

Comment:
Is there a reason why we use `max_level` instead of `finest_level` here?

Reply:
Same question when we call `PSATDBackwardTransformEB`.

Reply:
`finest_level` would be safer